04-func.md

```diff
@@ -739,3 +738,30 @@ but *do* have to provide `delimiter=` for the second parameter.
 >
 > print k
 > ~~~
+
+> ## Combining functions to use both loops, conditionals {.challenge}
+>
+> Motivation:
+> Imagine you have just returned from a week long holiday where you also totally blew your petrol budget. You realise it would be useful to know the
+> number of days where you went over your daily petrol usage allowance of $100. However, as you only recorded the miles travelled per day, and you know the
+> efficiency of your vehicle in litres/km, a few calculations are going to be needed. These new fangled python skills are going to come in handy.   
+>
+> Overall Goal:
+> Given a Python list of miles travelled per day, a measurement of litres per kilometre, and the price of fuel, calculate the number of days where you exceeded $100
+> 
+> ~~~ {.python}
+> trips = [80,50,170,180,80,50,200]
+> ~~~
+> Tasks:
+> 
+> 1. Use the above list of miles travelled per day in iPython and create a function that converts miles
+> to kilometres. 
+>
+> kilometres = miles / 1.61
+>
+> 2. Use a loop to display the results of calling the miles to kilometres function on each item in the trips list.
+> 
+> 3. Create another function that converts kilometres to the cost of that trip, given that the fuel efficiency of the car is 0.5 litres per kliometres, and fuel
```

Comment:
What about reword as

~~~
2.  Create another function that
    calculate the cost of that trip for each day
    and receive as arguments
    the list of kilometers traveled by day,
    the fuel efficiency of the car in liters per kilometers
    and the fuel cost per liter.

    To test your use 0.5 for the fuel efficiency and US$2 for the cost of fuel.
~~~